### PR TITLE
Add timer file

### PR DESCRIPTION
**Pull request by ElizaAlimasa**
Please review my file, Thank you!

### CODE REVIEW

File: javascript.js
Comment (Asebuque-ChrystineNoelle) [inline javascript.js:1]: The stop button on your output doesn't seem to align with its function, as it should reset the timer to 0, but instead, it only pauses it. I suggest that you should check again your function for the stop button.

**Review by Asebuque-ChrystineNoelle — CHANGES_REQUESTED** [review]
Kindly check my comment. Thank you!

File: javascript.js
Comment (ElizaAlimasa) [reply javascript.js:1]: I've resolved the issue. Could you please review it again? Thank you!

Comment (Asebuque-ChrystineNoelle) [reply javascript.js:1]: All is good and working well now.

**Review by Asebuque-ChrystineNoelle — CHANGES_REQUESTED** [review]

**Review by Asebuque-ChrystineNoelle — APPROVED** [review]
You can now merge your changes and upload your page in Netlify. Don't forget to update your README file that contains your Netlify URL. Thank you!

File: javascript.js
Comment (ElizaAlimasa) [reply javascript.js:1]: Thank you! ^-^

**Pull request merged**